### ISSUE TITLE
taskTree task stopped working with Gradle 6.8 release candidates

### ISSUE THREAD

**Issue by hkhc** [dorongold/gradle-task-tree#40]
Reproduce steps:
- Set up an arbitrary gradle projects, with Gradle 6.7.1 in `gradle-wrapper.properties`
- execute `./gradlew build taskTree` and get the task tree
- Change the gradle to version 6.8-rc4
- execute `./gradlew build taskTree` again
Expected:
- Showing task tree.
Actual:
- No task tree is shown, and the `build` task get executed.

**Comment by Smitty010** [comment]
Doesn't work in version 6.8 final (3/8/2021).  It's not just an issue in the rc release

**Comment by cblomqvist** [comment]
In Gradle 6.8 the taskTree task is always skipped:
```
> Task :taskTree UP-TO-DATE
Skipping task ':taskTree' as it has no actions.
```

**Comment by cblomqvist** [comment]
This hack in my `build.gradle` made it not up-to-date:
`tasks.taskTree.doFirst { System.currentTimeMillis() }`

But when running it still produces no output:
```
gradle tasktree mytask --info --warning-mode all
...
> Task :taskTree
Property 'taskDepth' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.8.3/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Property 'noRepeat' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.8.3/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
Caching disabled for task ':taskTree' because:
  Build cache is disabled
Task ':taskTree' is not up-to-date because:
  Task.upToDateWhen is false.
:taskTree (Thread[Execution worker for ':' Thread 2,5,main]) completed. Took 0.003 secs.
```

**Comment by dorongold** [comment]
Fixed by fb19b847e2a6c419092c2231cc1839377bfb675b.
